frontend/src/components/v2/RadioGroup/RadioGroup.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
Kindly avoid keeping non reusable components from components/v2. It should only contains reusable ui components. 

Ideally i would only keep a generic `RadioGroup` and `RadioGroupItem` as two separate components inside this file and then use it inside the heroku page.

I don't think this component will ever be used anywhere like this as its clearly containing heroku exclusive data

Reply:
Agree with this one; this should indeed contain generic components only.

Anyways, `RadioGroup` will not be used in this iteration of the Heroku integration update.